pkg/auth/api/api.go

```diff
@@ -493,6 +493,7 @@ func (s *authService) PrepareDemoUser() {
 						Id:          config.OrganizationId,
 						Name:        "Demo organization",
 						UrlCode:     "demo",
+						OwnerEmail:  "demo@demo.org",
```

Comment:
Let's implement this in the values.yaml as `organizationOwnerEmail`
Let's use `demo@bucketeer.io` as default.